article/app/pages/StoryHtmlPage.scala

```diff
@@ -12,7 +12,7 @@ import views.html.fragments.commercial.{blockthrough, pageSkin, survey}
 import views.html.fragments.page._
 import views.html.fragments.page.body._
 import views.html.fragments.page.head.stylesheets.{criticalStyleInline, criticalStyleLink, styles}
-import views.html.fragments.page.head.{fixIEReferenceErrors, headTag, titleTag, weAreHiring}
+import views.html.fragments.page.head.{orielScriptTag, _}
```

Comment:
Scala n00b question... do we need to import `orielScriptTag` explicitly if we're importing everything using the underscore?

Reply:
No we don't, but automatic IntelliJ thinks we do!